### DISCUSSION TITLE
Improved Tab scrolling

### DISCUSSION THREAD

**Discussion by jdsimcoe** [primer/github-vscode-theme#308] [Ideas]
I've added two tweaks to my `settings.json` that removes the Tab drop-shadow on scroll:

```json
"workbench.colorCustomizations": {
  "contrastBorder": "#00000000",
  "scrollbar.shadow": "#ffffff00",
},
```

**Result**
![image](https://user-images.githubusercontent.com/275617/84531588-a9628000-ac99-11ea-873d-69e386d1b186.png)

I've found that it gives the editor chrome a more minimal feel that aligns with the overall visual direction of the theme. I'm not sure if this is something you'll want to implement, so this is just a suggestion.

**Comment by orsenkucher** [comment]
Also shadow appears next to minimap when view is narrowed, but I kinda like it. Maybe we just should make it not so intense? @jdsimcoe

**Comment by orsenkucher** [comment]
haha, nevermind, I think removing shadow is a good change👍👍
